### PR TITLE
Travis fix

### PR DESCRIPTION
**Pull request by Khukhuna**
I dont know why, but I can't create a new issue, so I will write down here. 

after some research I found out that by default travis uses script: "npm test", We don't have tests written yet, so it was causing crash. :zap: 

I added basic assertion tests using **Mocha** & **Chai** (We should keep this oackages for future testing :smile:) , until we add some real tests.
It successfully passes the checks. :yum: 

**Build status of my fork:** [Build Status](https://travis-ci.com/Khukhuna/lead-management-server)

travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
 language: node_js
-
 sudo: enabled
+
+node_js:
+  - "7"
```
Comment (aashutoshrathi) [inline .travis.yml]: Make it `8`

File: test/app.test.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const assert = require("chai").assert;
```
Comment (aashutoshrathi) [inline test/app.test.js]: This works as temporary tests.
@nikitadhiman please check this test is okay for now.

**Comment by Khukhuna** [comment]
Oh, I accidently changed the name of package-lock :joy:
Did the request changes and reverted changing name of package-lock.json as well.
Thank you :smile:

File: test/app.test.js (hunk 2 above)
Comment (nikitadhiman) [reply test/app.test.js]: This is ok for now, but we may ask @Khukhuna to make more changes when needed. Thanks for your efforts thou

**Pull request merged**